docs/marks/axis.md

```diff
@@ -143,34 +143,22 @@ Plot.plot({
 ```
 :::
 
-You can emulate [Datawrapper’s time axes](https://blog.datawrapper.de/new-axis-ticks/) using `\n` (the line feed character) for multi-line tick labels, plus a bit of date math to detect the first month of each year.
+Time axes default to a consistent multi-line tick format, [à la Datawrapper](https://blog.datawrapper.de/new-axis-ticks/), for example showing the first month of each quarter, and the year:
 
 :::plot https://observablehq.com/@observablehq/plot-datawrapper-style-date-axis
 ```js
 Plot.plot({
+  x: {tickSpacing: 60},
```

Comment:
This example is a little less compelling having to customize the tick settings. (Presumably, `ticks: 30` or `ticks: d3.utcMonth.every(3)` would produce equivalent output?) Maybe that’s fine, but maybe there’s a dataset with a smaller domain that would produce multi-line ticks by default, resulting in a simpler example?

Reply:
we could do
`Plot.line(sftemp, Plot.windowY(7, {x: "date", y: "high"})).plot({y: {nice: true}})`